packages/eas-cli/src/build/configure.ts

```diff
@@ -102,5 +102,10 @@ async function createEasJsonAsync(projectDir: string): Promise<void> {
 
   await fs.writeFile(easJsonPath, `${JSON.stringify(easJson, null, 2)}\n`);
   await getVcsClient().trackFileAsync(easJsonPath);
-  Log.withTick(`Generated ${chalk.bold('eas.json')}`);
+  Log.withTick(
+    `Generated ${chalk.bold('eas.json')} ${learnMore(
+      'https://docs.expo.dev/build-reference/eas-json/',
+      { learnMoreMessage: '(learn more)' }
+    )}`
+  );
```

Comment:
```suggestion
  Log.withTick(
    `Generated ${chalk.bold('eas.json')}. ${learnMore('https://docs.expo.dev/build-reference/eas-json/')}`
  );
```

This is inconsistent with other places where we link to sth it's usually just "Learn more" or sth more specific e.g. "Learn more on customizing Metro"

Also, the current implementation would print
```
Generated eas.json (learn more): https://docs.expo.dev/build-reference/eas-json/
```
on terminals that do not support links, which does not look great

Reply:
@wkozyra95's suggestion will look good and be consistent with how we present this message elsewhere